Sources/Commands/SwiftPackageRegistryTool.swift

```diff
@@ -150,6 +140,221 @@ public struct SwiftPackageRegistryTool: ParsableCommand {
             }
         }
     }
+    
+    struct Login: SwiftCommand {
+        static let configuration = CommandConfiguration(
+            abstract: "Log in to a registry")
+
+        @OptionGroup(_hiddenFromHelp: true)
+        var globalOptions: GlobalOptions
+
+        @Argument(help: "The registry URL")
+        var url: String
+        
+        @Option(help: "Username")
+        var username: String?
+
+        @Option(help: "Password")
+        var password: String?
+        
+        @Option(help: "Access token")
+        var token: String?
```

Comment:
are token and password mutually exclusive? if so, it think argument parser allows us to mark them as such

Reply:
Either username with(out) password, or token